addons/web_tip/static/src/js/tip.js

```diff
@@ -260,10 +258,23 @@
             }
             return def;
         },
+ 
+        //if needed, scroll prior to displaying the tip
+        scroll_tip: function(){
```

Comment:
I think it would be more correct to name this function `scroll` or `scroll_to_tip`.